src/Cake.Common/Build/AzurePipelines/Data/AzurePipelinesPublishTestResultsData.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Cake.Core;
+using Cake.Core.IO;
+
+namespace Cake.Common.Build.AzurePipelines.Data
+{
+    /// <summary>
+    ///  Provides TF Build agent info for publishing test results
```

Comment:
Suggestion: `Description of test result information to publish to Azure Pipelines`